src/plugins/intel_cpu/src/onednn/iml_type_mapper.cpp

```diff
@@ -50,6 +50,7 @@ impl_desc_type parse_impl_name(std::string impl_desc_name) {
     SEARCH_WORD_2(nchw, ref);
     SEARCH_WORD_2(ncdhw, ref);
     SEARCH_WORD_2(wino, winograd);
+    SEARCH_WORD_2(brdgmm, brgconv);
 
```

Comment:
why not just use `brdgmm` instead of `brgconv`, to be consistent with oneDNN's naming convention and also helps debugging

Reply:
Done.